### PR TITLE
chore(rum): add eventtarget to the instrumentation types

### PR DESCRIPTION
**Pull request by ksivamuthu**
Add eventtarget to the instrumentation types.

### CODE REVIEW

**Review by vigneshshanmugam — APPROVED** [review]
Thanks @ksivamuthu

File: packages/rum/src/index.d.ts [hunk 1]
```diff
@@ -181,7 +181,9 @@ type LogLevel = 'trace' | 'debug' | 'info' | 'warn' | 'error'
 type TransactionEvents = 'transaction:start' | 'transaction:end'
 type InstrumentationTypes =
   | 'page-load'
+  | 'eventtarget'
   | 'error'
   | 'history'
   | 'fetch'
   | 'xmlhttprequest'
+
```
Comment (jahtalab) [inline packages/rum/src/index.d.ts:189]:
```suggestion
```

**Review by jahtalab — CHANGES_REQUESTED** [review]
Thanks @ksivamuthu ! 
would you please sign the CLA?
Also the lint task is failing, I've added a suggestion to fix that.

**Review by vigneshshanmugam — APPROVED** [review]
Will fix the commit message issue while merging.

**Pull request merged**